src/utils/index.js

```diff
@@ -0,0 +1,11 @@
+export const isExpired = (date) => {
```

Comment:
I try to avoid creating utils functions, see [this blog post](https://lostechies.com/chrismissal/2009/06/01/anti-patterns-and-worst-practices-utils-class/) for a set of good arguments on why.

Could you define this function in something like `src/state/metrics.js`? It seems to perform a similar task to `urls.js` in that folder.

Reply:
I did not know util functions were considered an anti pattern. Thank you for this great tip!

Reply:
Opinions on this topic do vary a bit (you will see `utils` functions in lots of packages and places), but I think the argument I presented above is pretty solid. 😄